src/controllers/manage/appointmentsSummaryController.js

```diff
@@ -59,5 +59,9 @@ angular.module('bahmni.appointments')
                 }
             };
 
+            $scope.changeFormat = function (date) {
+                return moment(date).format('D MMM, ddd');
```

Comment:
@supriyatw do we have any dateUtility file already there to move this method, so that it can be reused

Reply:
We do have a DateUtility file @binduak but it was present in the react code. Adding the method there might include tight coupling.